<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1286</width>
      <height>940</height>
     </rect>
    </property>
    <property name="text">
     <string>Subway</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>How many?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="people_number_lbl">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="people_group_1">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>130</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>1</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_2">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>220</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>2</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_3">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>310</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>3</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_4">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>410</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>4</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result_consol">
    <property name="geometry">
     <rect>
      <x>1295</x>
      <y>590</y>
      <width>181</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
